cmd/manager/main.go

```diff
@@ -113,6 +113,9 @@ func main() {
 		log.Info(err.Error())
 	}
 
+	// Strating event listener
+	go controller.EventListner()
```

Comment:
This seems to be out of band from the concept of controller/watcher.
This should be done within the controller code. 
IMO in this case it should be handled inside `chaosengine` controller